packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.routing.js

```diff
@@ -1,43 +1,103 @@
+import get from 'lodash/get';
+
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('app.hosting.cdn.order', {
     url: '/order',
     component: 'hostingCdnOrder',
     resolve: {
-      goBack: /* @ngInject */ goToHosting => goToHosting,
+      autoPayWithPreferredPaymentMethod: /* @ngInject */
```

Comment:
This sounds more like a function than a boolean

Reply:
It's the name as the parameter of the checkout API.
I've kept it unchanged